cookbooks/bcpc-hadoop/recipes/resource_manager.rb

```diff
@@ -22,7 +25,7 @@
   bash "create-hdfs-history-dir #{dir}" do
     code "hadoop fs -mkdir /user/history/#{dir} && hadoop fs -chmod 1777 /user/history/#{dir} && hadoop fs -chown yarn:mapred /user/history/#{dir}"
     user "hdfs"
-    not_if "sudo -u hdfs hadoop fs -test -d /user/history/#{dir}"
+   not_if "sudo -u hdfs hadoop fs -test -d /user/history/#{dir}"
```

Comment:
Corrected extraneous sudo's
Corrected indentation
Corrected `hadoop fs` to be `hdfs dfs`